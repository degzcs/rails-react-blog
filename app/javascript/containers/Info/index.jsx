import React from 'react';
import { Link } from "react-router-dom";

const Info = () => (
  <div className="vw-100 vh-100 primary-color d-flex align-items-center justify-content-center">
    <div className="jumbotron jumbotron-fluid bg-transparent">
      <div className="container secondary-color">
        <h1 className="display-4"> Blog: Rails-React app</h1>
        <p className="lead">
          created by Diego Gomez
        </p>
        <hr className="my-4" />
        <a
          href="https://github.com/degzcs/rails-react-blog"
          className="btn btn-lg custom-button"
          role="button"
          target='_blank'
        >
          Github Repo
        </a>
      </div>
    </div>
  </div>
);

export default Info;
