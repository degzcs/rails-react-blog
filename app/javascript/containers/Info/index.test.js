import React from "react";
import renderer from "react-test-renderer";
import { StaticRouter } from "react-router";

import Info from "./index.jsx";

test("checks rendering of Info component", () => {
  const component = renderer.create(
    <StaticRouter location="someLocation">
      <Info />
    </StaticRouter>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
