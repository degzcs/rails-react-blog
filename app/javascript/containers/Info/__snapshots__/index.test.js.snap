// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`checks rendering of Info component 1`] = `
<div
  className="vw-100 vh-100 primary-color d-flex align-items-center justify-content-center"
>
  <div
    className="jumbotron jumbotron-fluid bg-transparent"
  >
    <div
      className="container secondary-color"
    >
      <h1
        className="display-4"
      >
         Blog: Rails-React app
      </h1>
      <p
        className="lead"
      >
        created by Diego Gomez
      </p>
      <hr
        className="my-4"
      />
      <a
        className="btn btn-lg custom-button"
        href="https://github.com/degzcs/rails-react-blog"
        role="button"
        target="_blank"
      >
        Github Repo
      </a>
    </div>
  </div>
</div>
`;
